dashboard/app/models/pd/workshop.rb

```diff
@@ -74,6 +74,19 @@ class Pd::Workshop < ActiveRecord::Base
     ]
   }
 
+  COURSE_SHORT_NAME_MAP = {
+    COURSE_CSF => 'csf',
+    COURSE_CSP => 'csp',
+    COURSE_ECS => 'ecs',
+    COURSE_CS_IN_A => 'CSinA',
+    COURSE_CS_IN_S => 'CSinS',
+    COURSE_CSD => 'csd',
+    COURSE_COUNSELOR => 'ca',
+    COURSE_ADMIN => 'ca'
+  }
+
+  SECTION_TYPES = COURSE_SHORT_NAME_MAP.values.uniq.map{|name| "#{name}_workshop"}.freeze
```

Comment:
Is there a reason not to simply define `WORKSHOP_NAME_MAP` (rather than `COURSE_SHORT_NAME_MAP`) and avoid the cumbersomeness of `SECTION_TYPES`, `section_type`, etc.?